aptos-move/framework/src/natives/object.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Why not just use the existing create_signer via friend relation?

Reply:
It feels a little awkward, I didn't want object to depend on account... I think at some point in time account might depend on object. Or we'll have to make a new account module that knows both of them -- i.e., aptos account.

Reply:
This really should go in its own module... It's really ugly to have this duplicated code then.

Reply:
Hmm.... I guess that should be a compatible change...